src/Components/PresentationalComponents/TableColumns/RemediationColumn.js

```diff
@@ -0,0 +1,44 @@
+import React, { Fragment } from 'react';
+import PropType from 'prop-types';
+import { AnsibeTowerIcon } from '@patternfly/react-icons';import { Tooltip } from '@patternfly/react-core';
+import { FormattedMessage } from 'react-intl';
+import messages from '../../../Messages';
+import { OutlinedQuestionCircleIcon } from '@patternfly/react-icons';
+
+const RemediationColumn = ({ fixable }) => {
+    return (
+        <Fragment>
+            {fixable === true && <Fragment><AnsibeTowerIcon/> <FormattedMessage {...messages.playbook} /></Fragment>}
+            {fixable === false && <FormattedMessage {...messages.manual} />}
+            {
+
+                fixable ?? (
+                    <Fragment>
+                        <FormattedMessage {...messages.notAvailable} />
+                        <Tooltip exitDelay={1000} appendTo={document.querySelector('.vulnerability')} content={
```

Comment:
I propose using 2000ms exit delay, I found 1000ms way too short to click the link.

Reply:
I slightly disagree with that (it stays on for longer and it creates a weird effect when trying to hover other) but for consistency, i will change it to `2000` and it's up to UXD